user/security-in-qubes/vm-sudo.md

```diff
@@ -172,15 +172,7 @@ Do not rely on this for extra security.**
         auth sufficient pam_permit.so
         ```
 
-    - For Whonix, if prompts appear during boot, create `/etc/sudoers.d/zz99` and add these lines:
-
-        ```
-        ALL ALL=NOPASSWD: /usr/sbin/virt-what
-        ALL ALL=NOPASSWD: /usr/sbin/service whonixcheck restart
-        ALL ALL=NOPASSWD: /usr/sbin/service whonixcheck start
-        ALL ALL=NOPASSWD: /usr/sbin/service whonixcheck stop
-        ALL ALL=NOPASSWD: /usr/sbin/service whonixcheck status
-        ```
+    - For Whonix, if prompts appear during boot, [unsupported](https://forums.whonix.org/t/passwordless-root-prompt-at-boot/16370).
```

Comment:
This sentence appears to be incomplete (from a grammatical perspective, at least), so I don't understand exactly what this statement is attempting to convey. The thread you linked seems to say, "This [i.e., something related to passwordless root] is unsupported in Qubes, therefore it's also unsupported in Whonix." Is that right? If so, I'm not sure why this documentation page exists in the official documentation *at all*. Should I simply remove it (and perhaps move it to the community docs instead)?

CC: @marmarek

Reply:
> This sentence appears to be incomplete (from a grammatical perspective, at least), so I don't understand exactly what this statement is attempting to convey. The thread you linked seems to say, "This [i.e., something related to passwordless root] is unsupported in Qubes, therefore it's also unsupported in Whonix." Is that right?

Right.

> If so, I'm not sure why this documentation page exists in the official documentation _at all_. Should I simply remove it (and perhaps move it to the community docs instead)?

The question wasn't necessarily for me, however my view on it... I am not sure what to do with this page. The first chapter `Passwordless root access in qubes` seems to defend `/etc/sudoers.d/qubes` `user ALL=(ALL) NOPASSWD: ALL`. In the view of some, `WTF?! Have you lost your mind?!` (a citation from that file / website). That's not really user documentation. Advanced users and developers might wonder about this. And if it isn't explained, this might reflect bad on the reputation of Qubes. I guess that could be reason, kinda like like a mini FAQ why it was put there in the first place. On the other hand, advanced users / developers scrutinizing Qubes and looking at `/etc/sudoers.d/qubes` or the source code can already find the most up to date comment there. So I don't understand why it needs to be duplicated on the Qubes website.

Chapter `Replacing passwordless root access with Dom0 user prompt` is unsupported by Qubes. More like developer inspiration perhaps. Maybe a fit for community docs.

Reply:
> If so, I'm not sure why this documentation page exists in the official documentation at all. 

IMHO the most important reason for having this page, is to document what changes/configuration are part of the "passwordless root" feature. As you can see here, it isn't just sudo, so it's important to have it documented somewhere. At some point, having dom0 prompt for root access as one of supported configuration is something we'd like, but it requires some more work - for example to keep wifi password prompt working.

Reply:
Okay, thank you both for your input. It sounds like the original intention for this page was just to document passwordless root itself and the configuration changes associated with it. When the ability to replace passwordless root with a dom0 prompt was developed, this seemed like the natural place to document it. In retrospect, though, that turned out to be scope creep (for this doc page). It sounds like the right thing to do is to move this page to the developer docs and move the section on replacing passwordless root with a dom0 prompt to the community docs. I've opened a separate PR for that: #1296